### PR TITLE
Implement FHDA logger module

### PR DESCRIPTION
**Pull request by yifeili98**

### CODE REVIEW

**Comment by yifeili98** [comment]
CC      : (

File: src/FHDAlogger.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        origin: from which source this logger is from (for determining logging file name)
+        level: the level of the logger, options are [INFO, DEBUG, WARNING, ERROR, CRITICAL]
+    Raises:
+        KeyError: The level parameter is not valid
+    Returns:
+        the logger object
+
+    '''
+    level = level.upper()
+    if level == 'INFO':
```
Comment (kis87988) [inline src/FHDAlogger.py]: If we pass level directly to `basicConfig`, I think it also works, but will get exception

File: src/InsertData.py [hunk 2]
```diff
@@ -57,7 +52,7 @@ def check_file_open(filename):
     if filename:
         with open(filename, 'r') as file:
             return json.load(file)
-    raise FileNotFoundError('File', filename, 'is not found!')
+    raise FileNotFoundError('File ' + filename + ' is not found!')
```
Comment (kis87988) [inline src/InsertData.py]: LOL. We can use f-string

**Review by kis87988 — CHANGES_REQUESTED** [review]

**Comment by kis87988** [comment]
LGTM. Let's merge into this repo first. However, I am thinking we will have logger module to a repo. then our product have same format.

**Review by kis87988 — APPROVED** [review]

**Review by biankaiwen111 — APPROVED** [review]
LGTM QAQ

**Pull request merged**